map/point.mjs

```diff
@@ -0,0 +1,63 @@
+"use strict";
+export {Point};
+
+/**
+ * 2d point. Jas useful math methods like multiply, divide, etc.
+ * @param x x of point 
+ * @param y y of point
+ */
+function Point(x,y){
+    this.x = x;
+    this.y = y;
+}
+Point.prototype.toString = function(){
+    return "("+this.x+","+this.y+")";
+}
+/**
+ * Return a point that is the result of adding a scalar or point to this point.
+ * @param {*} object scalar or point to add
+ * @returns {Point} new point
+ */
+Point.prototype.add = function(object){
+    if(typeof(object) == "number"){
+        let asNum = parseFloat(object);
+        return new Point(this.x + asNum, this.y + asNum);
```

Comment:
All of these return new objects because modifying existing objects is usually a bad idea.  There's a good reason for it that I can't articulate right now.